src/main/java/ch/uzh/ifi/hase/soprafs21/service/ActivityService.java

```diff
@@ -75,8 +75,6 @@ public void setSwipingStatus(long activityId, String token, SwipeStatus swipeSta
         for (UserSwipeStatus userSwipeStatus : userSwipeStatusList){
             if (userSwipeStatus.getUser().getId().equals(user.getId())) {
                 userSwipeStatus.setSwipeStatus(swipeStatus);
-                userSwipeStatusRepository.save(userSwipeStatus);
```

Comment:
This reversed the change for the fix of the swiping. Is this on purpose?

Reply:
i will solve this